src/Raven.Server/Documents/Sharding/Handlers/ShardedDatabaseRequestHandler.cs

```diff
@@ -55,6 +55,16 @@ public Task<TResult> ExecuteSingleShardAsync<TResult>(JsonOperationContext conte
             return ShardExecutor.ExecuteSingleShardAsync(context, command, shardNumber, token);
         }
 
+        public async Task<TResult> ExecuteRemoteAsync<TResult>(RavenCommand<TResult> command, CancellationToken token = default)
+        {
+            var requestExecutor = DatabaseContext.AllNodesExecutor.GetRequestExecutorForNode(command.SelectedNodeTag);
+            using (requestExecutor.ContextPool.AllocateOperationContext(out JsonOperationContext ctx))
+            {
+                await requestExecutor.ExecuteAsync(command, ctx, token: token);
+                return command.Result;
+            }
+        }
+
```

Comment:
don't we have `ExecuteForNodeAsync` for that?

Reply:
Done